apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx

```diff
@@ -165,15 +183,28 @@ export function DashboardsIndexPage() {
                     />
                   </Button>
 
+                  <Button
+                    disabled={selectedItems.length !== 1}
+                    onClick={() => handleViewDashboard(selectedItems[0])}
+                  >
+                    <FormattedMessage
+                      defaultMessage="View"
+                      description="dashboards table header view button"
+                    />
+                  </Button>
+
                   <Button
                     data-testid={`table-create-dashboard`}
+                    className="btn-custom-primary"
```

Comment:
I believe className is deprecated in cloudscape. is there any other way to handle this or can we use what is provided by cloudscape?

Reply:
this is a class we have for the non existent orange button, this is not from cloudscape

Reply:
https://github.com/awslabs/iot-application/issues/1141

Reply:
Should we tackle this story first then as to not introduce tech debt into the package? It seems like this will be a common issue as we have more need to customize application beyond what is provided by cloudscape.

Reply:
As of now, no, we are not customizing anything except the primary button. Also, we need someone from our team to take a look, which will take time and blocking all the CRs happy minds have done so far until then does not makes sense. I dont think this is big enough issue, that we need to block 2-3 dev's work.